src/Raven.Client/Documents/Session/IRevisionsSessionOperations.cs

```diff
@@ -10,6 +10,19 @@
 
 namespace Raven.Client.Documents.Session
 {
+    
+    public enum ForceRevisionStrategy
```

Comment:
Let's rename this maybe to 'RevisionCreationStrategy'

Reply:
But why ? 
ASFAIK This doesn't apply to 'regular' revisions, only to Forced Revisions